packaging/machine-learning-api.spec

```diff
@@ -242,6 +242,7 @@ DESTDIR=%{buildroot} ninja -C build %{?_smp_mflags} install
 %files devel
 %{_includedir}/nnstreamer/nnstreamer.h
 %{_includedir}/nnstreamer/nnstreamer-single.h
+%{_libdir}/libcapi-nnstreamer.so.*
```

Comment:
What if developers try to install both `capi-machine-learning-inference` and `capi-machine-learning-inference-devel`, the conflict occurs. Is this your intent?

Reply:
`capi-machine-learning-inference-devel` package breaks if `capi-machine-learning-inference` is installed, and it will be so hard to find the reason why the devel package is not working for the developer.

So I think it's okay to let it conflict.

Reply:
This is reverted as suggested right below.